server/src/Core/Specifications/GameNameDuplicationsSpecification.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Ardalis.Specification;
+using BGS.ApplicationCore.Entities;
+
+namespace BGS.ApplicationCore.Specifications;
+
+public sealed class GameNameDuplicationsSpecification : SingleResultSpecification<Game, bool>
+{
+    public GameNameDuplicationsSpecification(string gameName)
+    {
+        Query.Where(game => string.Equals(game.Name.ToUpper(), gameName.ToUpper()));
```

Comment:
Let's think about our current situation. There cannot be two or more games with the same name in our application. So we need to configure restriction at the db level. Let's change type of name to `citext` and remove `.ToUpper()`